mysql-test/r/percona_utility_user.result

```diff
@@ -155,7 +155,7 @@ COUNT(*)
 0
 SELECT CURRENT_CONNECTIONS, TOTAL_CONNECTIONS FROM performance_schema.hosts WHERE HOST="localhost";
 CURRENT_CONNECTIONS	TOTAL_CONNECTIONS
-1	1
+2	2
```

Comment:
Why?

Reply:
This is a result of the following upstream change https://github.com/mysql/mysql-server/commit/4095e087545769c193bd0dc17b5449e5fbe17606